### ISSUE TITLE
Changing a zone's source should unmute zone.

### ISSUE THREAD

**Issue by linknum23** [micro-nova/AmpliPi#334]

**Comment by linknum23** [comment]
Yup, Also this would be in addition to #332 so the use case of moving a zone to a different source just to avoid playing to it will be handled there.

**Comment by rickballard** [comment]
And an "everything off" button, or swipe-shortcut, would be nice.  This would avoid a wife wanting to whack the AmpliPiSonoff wall-contrller with a frying pan...

**Comment by linknum23** [comment]
Planning on adding this to the wall panel already. Check out the related
issue here: https://github.com/micro-nova/AmpliPiWallPanel/issues/16

**Comment by rickballard** [comment]
Hah, I do not see an entry for a "whack-off gesture", what are you calling
this new feature?
*Rick Ballard  416.414.8100 *

**Comment by rickballard** [comment]
On the Amplipi Sonoff wall controller, can we have a less than zero notch,
on the volume slider, which folks will instinctively know means "mute
all"...?

Regards, Rick.  416.414.8100
